### PR TITLE
#133 - Player platform

### PR DESCRIPTION
**Pull request by EdgarVerona**

### CODE REVIEW

File: common/PlatformTypeUtils.gd [hunk 1]
```diff
@@ -0,0 +1,26 @@
+extends Node
+
```
Comment (Wavesonics) [inline common/PlatformTypeUtils.gd:3]: This should use `class_name PlatformTypeUtils` here instead of adding it to autoloads

File: project.godot [hunk 2]
```diff
@@ -197,6 +197,7 @@ vr="*res://addons/OQ_Toolkit/vr_autoload.gd"
 UserData="*res://common/UserData.gd"
 Maps="*res://common/game/maps/MapDirectory.gd"
 FugitivePlayerDataUtility="*res://common/game/mode/fugitive/networking/FugitivePlayerDataUtility.gd"
+PlatformTypeUtils="*res://common/PlatformTypeUtils.gd"
```
Comment (Wavesonics) [inline project.godot]: Remove this once the class declares it's own `class_name`

File: server/game/mode/fugitive/ServerFugitiveGame.gd [hunk 3]
```diff
@@ -41,7 +41,7 @@ func _ready():
 
 
 func load_map():
-	.load_map()
+	.load_map()
```
Comment (Wavesonics) [inline server/game/mode/fugitive/ServerFugitiveGame.gd]: clean up

**Review by Wavesonics — CHANGES_REQUESTED** [review]

File: common/PlatformTypeUtils.gd (hunk 1 above)
Comment (EdgarVerona) [reply common/PlatformTypeUtils.gd:3]: Oh, yes I keep forgetting about that!  Will do!

File: server/game/mode/fugitive/ServerFugitiveGame.gd (hunk 3 above)
Comment (EdgarVerona) [reply server/game/mode/fugitive/ServerFugitiveGame.gd]: Oh, I don't remember changing this file, good stuff!

**Review by Wavesonics — APPROVED** [review]

**Pull request merged**